app/src/main/kotlin/at/bitfire/davdroid/settings/migration/AccountSettingsMigration17.kt

```diff
@@ -40,6 +41,7 @@ class AccountSettingsMigration17 @Inject constructor(
 ): AccountSettingsMigration {
 
     override fun migrate(account: Account) {
+        val localAddressBookAccountUserDataUrl = "url"
```

Comment:
Good to keep the migration working!

But we can move it to a `const val` in the companion object, like every other String constant.